sa/_db-next/migrations/19700101000000_CombinedSchema.sql

```diff
@@ -0,0 +1 @@
+../../_db/migrations/19700101000000_CombinedSchema.sql
```

Comment:
Personally I'd avoid the file rename to `1970...`. It's only necessary when rolling back and forth across the commit which introduced the combined schema, which we don't do in our tests and which all of our local setups should be past by now. Maybe this would have been worth doing when I first introduced this file, to prevent any breakage when bisecting sometime in the future, but now that it already has the accurately-dated filename renaming it now won't fix that I don't think.

Reply:
Took a different approach in the new push. I have renamed all of the `_db-next` migrations so that they come after the combined schema in `_db`